backend_uml.go

```diff
@@ -10,6 +10,7 @@ import (
 	"os"
```

Comment:
IMHO the commit message would be cleaner if it covers the approach not the result. Namely:

On Arch a local version "-usermodelinux" is used, thus the modules are installed in `/usr/lib/modules/$version-usermodelinux". Effectively side-by-side to the normal modules.